src/css/custom.css

```diff
@@ -38,20 +38,37 @@ body {
   align-self: center;
 }
 
-/* laptops and up @media */
+/* @media laptops and up */
 @media screen
   and (min-width: 1200px) {
     .box-content {
-      max-width: 45%;
-      width: 45%;
+      max-width: 600px;
+      width: 600px;
```

Comment:
You're my only user feedback so far; get in early! **I can change it back, it's not a huge deal**. We didn't `max-width` the header, so in perspective everything else seems super narrow. However, controlled width makes readability (e.g. line length) and ease of focus that much easier on wider screens; variable for smaller. For instance, you won't see very many web apps using variable width on *everything*; there are defined content bounds.

I can change it back since our users probably won't have 50"-ers.